Source/FirebaseRemoteConfiguration.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  RemoteConfig.swift
+//  edX
+//
+//  Created by Salman on 04/11/2020.
+//  Copyright © 2020 edX. All rights reserved.
+//
+
+import UIKit
+
+fileprivate enum AppThemeKeys: String, RawStringExtractable {
+    case icon = "icon"
+    case font = "font"
+    case color = "color"
+    case mode = "mode"
+}
+
+fileprivate enum FontKeys: String, RawStringExtractable {
+    case name = "name"
+    case enable = "enabled"
+}
+
+fileprivate enum ColorKeys: String, RawStringExtractable {
+    case name = "name"
+    case enable = "enabled"
+}
+
+@objc class FirebaseRemoteConfiguration: NSObject {
+    @objc var appTheme: ThemeConfig?
+    @objc static let sharedRemoteConfig =  FirebaseRemoteConfiguration()
+    
+    @objc private override init() {
+        super.init()
+    }
```

Comment:
as `init` is called by `sharedRemoteConfig` to you can remove `@objc`

```suggestion
     var appTheme: ThemeConfig?
    @objc static let sharedRemoteConfig = FirebaseRemoteConfiguration()
    
    private override init() {
        super.init()
    }
```

Reply:
done